backend/db/resolve_fetch_failures_excess_items.py

```diff
@@ -78,7 +79,7 @@ def resolve_fetch_failures_excess_items(schema=SCHEMA, use_local_db=False, cache
                     if rows:
                         insert_from_dicts(con, dataset, rows)
                         solved_failures.append(failure)
-            refresh_derived_tables(con)
+            refresh_derived_tables(con, schema=schema)
```

Comment:
Bug fix: Part of `resolve_fetch_failures_excess_items()` did not include `schema` param, so wasn't running on test schema if requested.